src/main/java/com/marafone/marafone/game/model/GameDTO.java

```diff
@@ -0,0 +1,4 @@
+package com.marafone.marafone.game.model;
+
+public record GameDTO(String gameName, GameType gameType, Integer joinedPlayersAmount) {
```

Comment:
User need to have a gameId in order to join the lobby.
https://github.com/Marafone/Backend/blob/ba74de9c4c34bc90e28a7b336a2efa81294050b4/src/main/java/com/marafone/marafone/game/active/ActiveGameController.java#L44
That means that GameDTO should return id (at which it is stored in hashmap) in order to let users join the game.
That id was stored in the "name" field in Game class, but you changed that in createGame method in ActiveGameServiceImpl. Keep in mind that  id field in Game class stores the id that is used in realtional database (after the games ends, game is saved to relational database and this id is autogenerated) which is diffrent from the id used in hashmap, so in order for GameDto to return the id from hashmap you probably need to add additional field in Game with @Transient annotation. (Or you could force hashmap and relational database to use same id, but I think that would be harder)